sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn

```diff
@@ -652,532 +829,344 @@ dmnX (comes from dmn)</bpmn:text>
-      </bpmn:extensionElements>
-      <bpmn:incoming>Flow_1xpxz73</bpmn:incoming>
-      <bpmn:outgoing>Flow_1dy9nx0</bpmn:outgoing>
-    </bpmn:scriptTask>
-    <bpmn:scriptTask id="Activity_1r39i7t" name="Dependants">
-      <bpmn:extensionElements>
-        <zeebe:script expression="=[]" resultVariable="calculatedDataDependants" />
-      </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0hgiad0</bpmn:incoming>
-      <bpmn:outgoing>Flow_0riozxs</bpmn:outgoing>
-    </bpmn:scriptTask>
-    <bpmn:exclusiveGateway id="Gateway_0m9brpo" default="Flow_0hgiad0">
-      <bpmn:incoming>Flow_0z4iwi4</bpmn:incoming>
-      <bpmn:outgoing>Flow_1xpxz73</bpmn:outgoing>
-      <bpmn:outgoing>Flow_0hgiad0</bpmn:outgoing>
-    </bpmn:exclusiveGateway>
-    <bpmn:exclusiveGateway id="Gateway_0y2ax1o">
-      <bpmn:incoming>Flow_1dy9nx0</bpmn:incoming>
-      <bpmn:incoming>Flow_0riozxs</bpmn:incoming>
-      <bpmn:outgoing>Flow_0a0yhpv</bpmn:outgoing>
-    </bpmn:exclusiveGateway>
-    <bpmn:sequenceFlow id="Flow_1xpxz73" name="Has Dependants" sourceRef="Gateway_0m9brpo" targetRef="Activity_0v4suzm">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=calculatedDataHasDependents = "yes"</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="Flow_1dy9nx0" sourceRef="Activity_0v4suzm" targetRef="Gateway_0y2ax1o" />
-    <bpmn:sequenceFlow id="Flow_0hgiad0" name="Has No Dependants" sourceRef="Gateway_0m9brpo" targetRef="Activity_1r39i7t" />
-    <bpmn:sequenceFlow id="Flow_0riozxs" sourceRef="Activity_1r39i7t" targetRef="Gateway_0y2ax1o" />
-    <bpmn:sequenceFlow id="Flow_0z4iwi4" sourceRef="Gateway_08rr1xe" targetRef="Gateway_0m9brpo" />
-    <bpmn:sequenceFlow id="Flow_0a0yhpv" sourceRef="Gateway_0y2ax1o" targetRef="Event_0hxxmu9" />
-    <bpmn:scriptTask id="Activity_1vwijyc" name="Dependants">
+    <bpmn:sequenceFlow id="Flow_0ra2rum" sourceRef="Event_0ip2nw4" targetRef="Event_10lg4rt" />
+    <bpmn:intermediateThrowEvent id="Event_10lg4rt" name="Calculate total transportation Allowance">
       <bpmn:extensionElements>
-        <zeebe:script expression="={&#10;  calculatedDataHasDependents: studentDataHasDependents,&#10;  calculatedDataDependants: (if studentDataDependants = null then null else studentDataDependants)&#10;}" resultVariable="calculatedDataDependantDetails" />
         <zeebe:ioMapping>
-          <zeebe:output source="=calculatedDataDependantDetails.calculatedDataHasDependents" target="calculatedDataHasDependents" />
-          <zeebe:output source="=calculatedDataDependantDetails.calculatedDataDependants" target="calculatedDataDependants" />
+          <zeebe:output source="=if(offeringDelivered = &#34;onsite&#34;)&#10;then &#10;  offeringWeeks *  dmnPartTimeProgramYearMaximums.limitTransportationAllowance&#10;else&#10;  0" target="calculatedDataTotalTransportationAllowance" />
```

Comment:
Minor, there is an additional white space in the below expression.
```
offeringWeeks *  dmnPartTimeProgramYearMaximums.limitTransportationAllowance
```

Reply:
This is so minor that would change only if some other change is needed, otherwise, I would recommend to leave it as it is.